TrackForce/src/main/java/com/revature/utils/ThreadUtil.java

```diff
@@ -24,6 +24,7 @@ public ThreadUtil() {
 		
 		try {
 			results = future.get();
+			return results;
```

Comment:
I don't think this additional return is necessary, and it bypasses a log statement. Is that intentional?